packages/frontend/src/components/attachment/messageAttachment.tsx

```diff
@@ -217,6 +218,19 @@ export function DraftAttachment({
     )
   } else if (isAudio(attachment.fileMime)) {
     return <AudioPlayer src={runtime.transformBlobURL(attachment.file || '')} />
+  } else if (attachment.webxdcInfo) {
+    const iconUrl = runtime.getWebxdcIconURL(selectedAccountId(), attachment.id)
+    return (
+      <div className='media-attachment-webxdc'>
```

Comment:
The classes that this uses are defined in the `gallery` directory. Not super nice, but works for now I guess.